### ISSUE TITLE
Cannot access 'KoinTest' when updating from 3.1.6 to 3.2.0

### ISSUE THREAD

**Issue by rohte** [InsertKoinIO/koin#1415]
**Describe the bug**
The `KoinTest` class is not accessible any more for version 3.2.0. We have some tests that inherit from `AutoCloseKoinTest` and on update they create the error
```
Cannot access 'org.koin.test.KoinTest' which is a supertype of 'TestClass'. Check your module classpath for missing or conflicting dependencies
```
It is also impossible to import `KoinTest` itself after updating.

**To Reproduce**
1. Have these dependencies in your gradle:
```
implementation("io.insert-koin:koin-core:3.1.6")
testImplementation("io.insert-koin:koin-test-junit5:3.1.6")
```
2. Write a test that inherits from `org.koin.test.junit5.AutoCloseKoinTest`
3. Update Koin version in gradle to 3.2.0

**Expected behavior**
The AutoCloseKoinTest has no dependency problems. 

**Koin project used and used version (please complete the following information):**
`koin-core version 3.2.0`
`koin-test-junit5 version 3.2.0`

**Comment by arnaudgiuliani** [comment]
yes, there is a metadata glitch here. You can add `testImplementation("io.insert-koin:koin-test:3.1.6")`

**Comment by arnaudgiuliani** [comment]
metadata are fixed in `3.2.1`
